tasks/task1155_bard_analogical_reasoning_trash_or_treasure.json

```diff
@@ -0,0 +1,3324 @@
+{
+    "Contributors": [
+        "Arjun Ashok"
+    ],
+    "Source": [
+        "https://github.com/NancyFulda/BYU-Analogical-Reasoning-Dataset"
+    ],
+    "Definition": "Two analogies that relate items to whether they are trash or treasure is given in the form \"A : B. C : ?\". \"A : \"B relates item A to whether it is trash or treasure, as specified by B. Your task is to replace the question mark (?) with either `trash` or `treasure` for the given item C, following the \"A : B\" relation.",
```

Comment:
not quite sure if I understand this task.

Reply:
You need to properly define the task, by defining what it means to be "trash" or "treasure": 

 - trash ->   should be disposed of
 - treasure -> it a precious item which must be retained

Reply:
@danyaljj Done. Can this be merged now?